tv_apps/tv-epg/style/tv_epg.css

```diff
@@ -28,8 +28,8 @@ a:hover, a:active, a:focus {
 }
 
 #main-container {
-  width: 100%;
-  height: 100%;
+  width: 192rem;
```

Comment:
Why do you change the width/height to a fixed value instead of 100%? Although our target device is 1080p, we should still keep it resizable.